mypy	-0.03179631951519089
backport	-0.021813423852672657
acb2eadc8e692be4c8212cf1d3697e10e298a384	-0.016370216315872238
Python	-0.0064681433684105364
subprocess	-0.00646598077882661
bug	-0.005774352328195181
run	0.005258077259058875
picked	-0.0046234490367945595
The	-0.004049361909149627
Specify	-0.0036996579470937284
setup	-0.0034697530940140265
version	-0.0032474805344497857
fix	-0.002663671262197419
com	-0.0026252706380355518
Subject	-0.002588261662247592
Conflicts	-0.002578270237813161
caught	0.002567635930797801
target	-0.002317452616348036
encoding	0.002317289521342411
pull	0.0023016509340761473
function	-0.002300698863213221
with	-0.0022809839613169435
https	-0.002223577387377865
But	0.0015043255051341382
on	0.0014008948841835297
